### DISCUSSION TITLE
Pipe position simulation ?

### DISCUSSION THREAD

**Discussion by aleph00** [GrandOrgue/grandorgue#1010] [Help]
Hi all,

I discovered GO a few weeks ago to modernize my old viscount organ. I running it with the Freisach samples and I'm really amazed by the results ! You guys are doing a wonderful job ! 

One of the thing I love when playing a real pipe organ is to hear the music dancing around me as the pipes sing, each from its own location : behind me, in front of me, on the right, the left, it's just amazing :-) We can't hear that from down the church, but from the console it's just great.

I'm wondering if we could reproduce this with GO and a 5.1 sound card ? The left/right can be done by tweaking the samples, but how to do the front/rear thing ? The only way I've found is to define an audio group for each rank, then assign those audio groups eg 30% on front channels and 70% on back channels playing with the dB in audio settings. But that's going to be a *lot* of audio groups (44 for Freisach) and not really nice as organ properties are being set in the audio settings. 

Is it the way to go or is there another possibility ?

**Comment by oleg68** [comment]
> The only way I've found is to define an audio group for each rank, then assign those audio groups eg 30% on front channels and 70% on back channels playing with the dB in audio settings. But that's going to be a _lot_ of audio groups (44 for Freisach) and not really nice as organ properties are being set in the audio settings.

Do you really need to have a separate audio group for each rank? Usually several ranks are placed on the same windlade, so the pipes from several stops are very close to each other and do not require separate positioning. So you can group several ranks to one audio group. In this case you have less audio groups.

**Reply by aleph00** [reply]
Ah yes, good idea :-) I'll give this a try ; thanks for the hint
